GoogleForCreators/web-stories-wp
# Shopping: Floating Menu

## Context

Add a floating menu with product dropdown for the selected product.

[Figma](https://www.figma.com/file/ZrO763yvMWJjYeAPsRnNfW/_Editor---Sprint-Feature-Level-Designs_2022?node-id=503%3A15921)

https://github.com/GoogleForCreators/web-stories-wp/issues/11252#issue-1201793703



## Summary

- Moves the current product dropdown into a component for re-use in the floating menu bar
- Adds floating menu bar
- Add trash button to remove the product



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Turn on the shopping feature (ensure you have some products available)
2. Add a story with a product 
3. Click on the product on the stage
4. Floating menu should appear with the product dropdown + trash button
5. Select a different product --- this should update the existing product
6. Delete the product using the `Trash` button


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #11252
